page/jquery-ui/widget-factory/classes-option.md

```diff
@@ -0,0 +1,134 @@
+<script>{
+	"title": "Using the classes Option",
+	"level": "advanced"
+}</script>
+
+As of the 1.12 release, the jQuery UI widget factory includes a means of managing CSS class names through the [`classes` option](http://api.jqueryui.com/jquery.widget/#option-classes). This article will give you an overview of how the `classes` option works, and discuss what you can do with it.
+
+## Syntax overview
+
+The `classes` option is used to map structural class names to custom class names that you define. To see what this means let's look at an example. The code below uses the `classes` option to create a red dialog:
```

Comment:
This is not entirely true it maps structural class names to theme class's ( both custom and built in ). You are also doing more then just making the dialog red here you are also removing the rounded corners.